lib/index.js

```diff
@@ -33,23 +36,17 @@ document.addEventListener('keydown', function(event){
   if (event.keyCode === 37) { world.leftArrow();}
 });
 
-function drawAllJunkFoods(junkFoods){
-  for (var i = 0; i < junkFoods.length; i++){
-    var junkFood = junkFoods[i];
-    junkFoodDraw(junkFoods[i]);
-    junkFood.move();
-    if(junkFoods[i].y > 599){
-      junkFoods.shift();
-    }
+function drawFoods(foods){
```

Comment:
since all of our functions relating to foods are more generic now, maybe we should just have one general food constructor?